pkg/nsx/client.go

```diff
@@ -41,12 +51,21 @@ func GetClient(cf *config.NSXOperatorConfig) *Client {
 	groupClient := domains.NewGroupsClient(restConnector(cluster))
 	securityClient := domains.NewSecurityPoliciesClient(restConnector(cluster))
 	ruleClient := security_policies.NewRulesClient(restConnector(cluster))
+	nsxChecker := func(req *http.Request) error {
```

Comment:
Could we define a separate function in case that we may add more logic for health check later